ci-operator/step-registry/openstack-k8s-operators/gather/openstack-k8s-operators-gather-commands.sh

```diff
@@ -1,21 +1,64 @@
 #!/usr/bin/env bash
 
 set -x
+set +e
+
+BASE_DIR=${HOME:-"/alabama"}
+MUST_GATHER_TIMEOUT=${MUST_GATHER_TIMEOUT:-"5m"}
+
+function create_out_dir {
+  local OUT_DIR=${1}
+  if [ ! -d "./${OUT_DIR}" ]; then
+    mkdir ${OUT_DIR}
+  fi
+}
+
+function gen_pods_debug_cmd {
+  create_out_dir ${1}
+  local cmd=("echo 'Logs for pod: {}'; oc describe pod {} > ${1}/{}-describe.log; oc logs --all-containers=true {} >> ${1}/{}-containers.log")
+  echo "${cmd[@]}"
+}
+
+function gen_api_resources_debug_cmd {
+  create_out_dir ${1}
+  local cmd=("echo 'Info for api-resource: {}'; oc get {} -o yaml > ${1}/{}.log")
```

Comment:
should this be a `.yaml` instead of `.log`

Reply:
make totally sense, i will update that